### ISSUE TITLE
js on the server

### ISSUE THREAD

**Issue by joshua1** [feathers-nuxt/template-app#5]
can a js option for the backend be added to the application so that users have the option of picking livescript while other take JS or TS

**Comment by kharhys** [comment]
Yes, the next release will introduce an option for picking preferred language for back end. LS and JS will be available. TS may be added in future releases.
